challenges/functions.js

```diff
@@ -6,28 +6,33 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
+const consume = ((param1, param2, cb) => {return cb(param1, param2);})
```

Comment:
`let consume = (param1, param2, cb) => callback(a, b);`

For => syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions